docs/commands.md

```diff
@@ -2,6 +2,14 @@
 
 Harpoon has several commands that you can use.
 
+The common way of using `harpoon` is to execute the available commands as follow:
+
+* [`harpoon analyze`](#analyze-) to analyze the project to infer symbols to be traced. This will create a `.harpoon.yml` file.
+
+* [`harpoon hunt`](#hunt-) by passing the `.harpoon.yml` file to trace the functions and get their system calls. This will generate the `./harpoon/` directory with the metadata that contains the system calls traced.
```

Comment:
```suggestion
* [`harpoon hunt`](#hunt-) by passing the `.harpoon.yml` file to trace the functions and get their system calls. This will generate the `./harpoon/` directory with the metadata that contain the system calls traced.
```

I would say that metadata is always plural.
it's worth to check